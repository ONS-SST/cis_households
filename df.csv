,ons_household_id,participant_id,sex,ethnicity_white,age_at_visit,index_multiple_deprivation,sample_addressbase_indicator,country_name_12,region_code,cis_area_code_20,household_level_designweight_swab,household_level_designweight_antibodies,swab,antibodies,longcovid,ever_never,14_days,28_days,42_days,number_of_households_population_by_cis,interim_participant_id,index_multiple_deprivation_group,check_if_missing,response_indicator,population_country_swab,population_country_antibodies,total_sampled_households_cis_imd_addressbase,total_responded_households_cis_imd_addressbase,raw_non_response_factor,mean_raw_non_response_factor,scaled_non_response_factor,bounded_non_response_factor,household_level_designweight_adjusted_swab,household_level_designweight_adjusted_antibodies,sum_adjusted_design_weight_antibodies,scaling_factor_adjusted_design_weight_antibodies,scaled_design_weight_adjusted_antibodies,sum_adjusted_design_weight_swab,scaling_factor_adjusted_design_weight_swab,scaled_design_weight_adjusted_swab,interim_region_code,interim_sex,age_group_swab,age_group_antibodies,swab_evernever,swab_14days,longcovid_24days,longcovid_42days,antibodies_evernever,antibodies_28daysto,p1_swab_longcovid_england,p1_swab_longcovid_wales_scot_ni,p1_for_antibodies_evernever_engl,p1_for_antibodies_28daysto_engl,p1_for_antibodies_wales_scot_ni,p2_for_antibodies,p3_for_antibodies_28daysto_engl
0,6,15.0,male,0.0,50,800,12,northern_ireland,E12000006,12,0.9,1.1,1,1,1,1,1,1,1,2,1.0,,,1.0,600,700,1,1,1.0,1.0,1.0,,,,,,,,,,6,1.0,6,4.0,1,,1,1,1,1,,6.0,,,4.0,,
1,6,14.0,male,1.0,50,800,11,northern_ireland,E12000006,11,0.9,1.1,1,1,1,1,1,1,1,2,1.0,,,1.0,600,700,1,1,1.0,1.0,1.0,,,,,,,,,,6,1.0,6,4.0,1,,1,1,1,1,,6.0,,,4.0,,
2,7,16.0,male,0.0,50,800,13,england,E12000007,13,0.9,1.1,1,1,1,1,1,1,1,17,,1.0,,1.0,1500,1200,5,4,1.3,1.176,1.105,,,,,,,,,,7,1.0,6,4.0,1,1.0,1,1,1,1,90.0,,64.0,4.0,,1.0,7.0
3,7,17.0,male,0.0,50,800,13,england,E12000007,13,0.9,1.1,1,1,1,1,1,1,1,17,,1.0,,1.0,1500,1200,5,4,1.3,1.176,1.105,,,,,,,,,,7,1.0,6,4.0,1,1.0,1,1,1,1,90.0,,64.0,4.0,,1.0,7.0
4,7,19.0,male,0.0,50,800,13,england,E12000007,13,0.9,1.1,1,1,1,1,1,1,1,17,,1.0,,1.0,1500,1200,5,4,1.3,1.176,1.105,,,,,,,,,,7,1.0,6,4.0,1,1.0,1,1,1,1,90.0,,64.0,4.0,,1.0,7.0
5,7,20.0,male,0.0,50,800,13,england,E12000007,13,0.9,1.1,1,1,1,1,1,1,1,17,,1.0,,1.0,1500,1200,5,4,1.3,1.176,1.105,,,,,,,,,,7,1.0,6,4.0,1,1.0,1,1,1,1,90.0,,64.0,4.0,,1.0,7.0
6,7,18.0,male,0.0,50,800,13,england,E12000007,13,0.9,1.1,1,1,1,1,1,1,1,17,1.0,1.0,,1.0,1500,1200,5,4,1.3,1.176,1.105,,,,,,,,,,7,1.0,6,4.0,1,1.0,1,1,1,1,90.0,,64.0,4.0,,1.0,7.0
7,1,1.0,male,1.0,50,7000,1,england,E12000001,1,0.9,1.1,1,1,1,1,1,1,1,17,1.0,2.0,,1.0,1500,1200,1,1,1.0,1.176,0.85,,,,,,,,,,1,1.0,6,4.0,1,1.0,1,1,1,1,6.0,,4.0,4.0,,2.0,1.0
8,5,10.0,male,1.0,50,7000,7,england,E12000005,7,0.9,1.1,1,1,1,1,1,1,1,17,1.0,2.0,,1.0,1500,1200,4,4,1.0,1.176,0.85,,,,,,,,,,5,1.0,6,4.0,1,1.0,1,1,1,1,62.0,,44.0,4.0,,2.0,5.0
9,5,11.0,male,1.0,50,7000,7,england,E12000005,7,0.9,1.1,1,1,1,1,1,1,1,17,1.0,2.0,,1.0,1500,1200,4,4,1.0,1.176,0.85,,,,,,,,,,5,1.0,6,4.0,1,1.0,1,1,1,1,62.0,,44.0,4.0,,2.0,5.0
10,5,12.0,male,1.0,50,7000,7,england,E12000005,7,0.9,1.1,1,1,1,1,1,1,1,17,1.0,2.0,,1.0,1500,1200,4,4,1.0,1.176,0.85,,,,,,,,,,5,1.0,6,4.0,1,1.0,1,1,1,1,62.0,,44.0,4.0,,2.0,5.0
11,5,13.0,male,1.0,50,7000,7,england,E12000005,7,0.9,1.1,1,1,1,1,1,1,1,17,1.0,2.0,,1.0,1500,1200,4,4,1.0,1.176,0.85,,,,,,,,,,5,1.0,6,4.0,1,1.0,1,1,1,1,62.0,,44.0,4.0,,2.0,5.0
12,2,4.0,female,0.0,30,6500,3,england,E12000002,3,0.9,1.1,1,1,1,1,1,1,1,17,1.0,1.0,,1.0,1500,1200,4,4,1.0,1.176,0.85,,,,,,,,,,2,2.0,4,2.0,1,1.0,1,1,1,1,25.0,,17.0,7.0,,1.0,2.0
13,2,5.0,male,1.0,32,6500,3,england,E12000002,3,0.9,1.1,1,1,1,1,1,1,1,17,1.0,1.0,,1.0,1500,1200,4,4,1.0,1.176,0.85,,,,,,,,,,2,1.0,4,2.0,1,1.0,1,1,1,1,18.0,,12.0,2.0,,2.0,2.0
14,2,6.0,male,0.0,5,6500,3,england,E12000002,3,0.9,1.1,1,1,1,1,1,1,1,17,1.0,1.0,,1.0,1500,1200,4,4,1.0,1.176,0.85,,,,,,,,,,2,1.0,1,,1,1.0,1,1,1,1,15.0,,,,,1.0,
15,2,7.0,male,0.0,3,6500,3,england,E12000002,3,0.9,1.1,1,1,1,1,1,1,1,17,1.0,1.0,,1.0,1500,1200,4,4,1.0,1.176,0.85,,,,,,,,,,2,1.0,1,,1,1.0,1,1,1,1,15.0,,,,,1.0,
16,3,8.0,male,1.0,2,15000,5,england,E12000003,5,0.9,1.1,1,1,1,1,1,1,1,17,1.0,3.0,,1.0,1500,1200,3,2,1.5,1.176,1.276,,,,,,,,,,3,1.0,1,,1,1.0,1,1,1,1,29.0,,,,,2.0,
17,3,8.0,,,2,15000,5,england,E12000003,5,0.9,1.1,1,1,1,1,1,1,1,17,1.0,3.0,1.0,1.0,1500,1200,3,2,1.5,1.176,1.276,,,,,,,,,,3,,1,,1,1.0,1,1,1,1,,,,,,,
18,3,,male,1.0,2,15000,5,england,E12000003,5,0.9,1.1,1,1,1,1,1,1,1,17,,3.0,1.0,,1500,1200,3,2,1.5,1.176,1.276,,,,,,,,,,3,1.0,1,,1,1.0,1,1,1,1,29.0,,,,,2.0,
19,4,9.0,male,1.0,30,400,6,wales,E12000004,6,0.9,1.1,1,1,1,1,1,1,1,1,1.0,2.0,,1.0,900,800,1,1,1.0,1.0,1.0,,,,,,,,,,4,1.0,4,2.0,1,,1,1,1,1,,4.0,,,2.0,2.0,
