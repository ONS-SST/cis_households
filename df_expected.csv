,ons_household_id,participant_id,sex,ethnicity_white,age_at_visit,index_multiple_deprivation,sample_addressbase_indicator,country_name_12,region_code,cis_area_code_20,household_level_designweight_swab,household_level_designweight_antibodies,check_if_missing,swab,antibodies,longcovid,ever_never,14_days,28_days,42_days,number_of_households_population_by_cis,response_indicator,interim_participant_id,index_multiple_deprivation_group,population_country_swab,population_country_antibodies,total_sampled_households_cis_imd_addressbase,total_responded_households_cis_imd_addressbase,raw_non_response_factor,mean_raw_non_response_factor,scaled_non_response_factor,bounded_non_response_factor,household_level_designweight_adjusted_swab,household_level_designweight_adjusted_antibodies,sum_adjusted_design_weight_swab,scaling_factor_adjusted_design_weight_swab,scaled_design_weight_adjusted_swab,sum_adjusted_design_weight_antibodies,scaling_factor_adjusted_design_weight_antibodies,scaled_design_weight_adjusted_antibodies,interim_region_code,interim_sex,age_group_swab,age_group_antibodies,p1_swab_longcovid_england,p1_swab_longcovid_wales_scot_ni,p1_for_antibodies_evernever_engl,p1_for_antibodies_28daysto_engl,p1_for_antibodies_wales_scot_ni,p2_for_antibodies,p3_for_antibodies_28daysto_engl,antibodies_28daysto,antibodies_evernever,longcovid_24days,longcovid_42days,swab_14days,swab_evernever
0,1,1.0,male,1.0,50,7000,1,england,E12000001,1,0.9,1.1,,1,1,1,1,1,1,1,17,1.0,1.0,2.0,1500,1200,1,1,1.0,1.176,0.85,,,,,,,,,,1,1.0,6,4.0,6.0,,4.0,4.0,,2.0,1.0,1,1,1,1,1.0,1
1,2,4.0,female,0.0,30,6500,3,england,E12000002,3,0.9,1.1,,1,1,1,1,1,1,1,17,1.0,1.0,1.0,1500,1200,4,4,1.0,1.176,0.85,,,,,,,,,,2,2.0,4,2.0,25.0,,17.0,7.0,,1.0,2.0,1,1,1,1,1.0,1
2,2,5.0,male,1.0,32,6500,3,england,E12000002,3,0.9,1.1,,1,1,1,1,1,1,1,17,1.0,1.0,1.0,1500,1200,4,4,1.0,1.176,0.85,,,,,,,,,,2,1.0,4,2.0,18.0,,12.0,2.0,,2.0,2.0,1,1,1,1,1.0,1
3,2,6.0,male,0.0,5,6500,3,england,E12000002,3,0.9,1.1,,1,1,1,1,1,1,1,17,1.0,1.0,1.0,1500,1200,4,4,1.0,1.176,0.85,,,,,,,,,,2,1.0,1,,15.0,,,,,1.0,,1,1,1,1,1.0,1
4,2,7.0,male,0.0,3,6500,3,england,E12000002,3,0.9,1.1,,1,1,1,1,1,1,1,17,1.0,1.0,1.0,1500,1200,4,4,1.0,1.176,0.85,,,,,,,,,,2,1.0,1,,15.0,,,,,1.0,,1,1,1,1,1.0,1
5,3,,male,1.0,2,15000,5,england,E12000003,5,0.9,1.1,1.0,1,1,1,1,1,1,1,17,,,3.0,1500,1200,3,2,1.5,1.176,1.276,,,,,,,,,,3,1.0,1,,29.0,,,,,2.0,,1,1,1,1,1.0,1
6,3,8.0,male,1.0,2,15000,5,england,E12000003,5,0.9,1.1,,1,1,1,1,1,1,1,17,1.0,1.0,3.0,1500,1200,3,2,1.5,1.176,1.276,,,,,,,,,,3,1.0,1,,29.0,,,,,2.0,,1,1,1,1,1.0,1
7,3,8.0,,,2,15000,5,england,E12000003,5,0.9,1.1,1.0,1,1,1,1,1,1,1,17,1.0,1.0,3.0,1500,1200,3,2,1.5,1.176,1.276,,,,,,,,,,3,,1,,,,,,,,,1,1,1,1,1.0,1
8,4,9.0,male,1.0,30,400,6,wales,E12000004,6,0.9,1.1,,1,1,1,1,1,1,1,1,1.0,1.0,2.0,900,800,1,1,1.0,1.0,1.0,,,,,,,,,,4,1.0,4,2.0,,4.0,,,2.0,2.0,,1,1,1,1,,1
9,5,10.0,male,1.0,50,7000,7,england,E12000005,7,0.9,1.1,,1,1,1,1,1,1,1,17,1.0,1.0,2.0,1500,1200,4,4,1.0,1.176,0.85,,,,,,,,,,5,1.0,6,4.0,62.0,,44.0,4.0,,2.0,5.0,1,1,1,1,1.0,1
10,5,11.0,male,1.0,50,7000,7,england,E12000005,7,0.9,1.1,,1,1,1,1,1,1,1,17,1.0,1.0,2.0,1500,1200,4,4,1.0,1.176,0.85,,,,,,,,,,5,1.0,6,4.0,62.0,,44.0,4.0,,2.0,5.0,1,1,1,1,1.0,1
11,5,12.0,male,1.0,50,7000,7,england,E12000005,7,0.9,1.1,,1,1,1,1,1,1,1,17,1.0,1.0,2.0,1500,1200,4,4,1.0,1.176,0.85,,,,,,,,,,5,1.0,6,4.0,62.0,,44.0,4.0,,2.0,5.0,1,1,1,1,1.0,1
12,5,13.0,male,1.0,50,7000,7,england,E12000005,7,0.9,1.1,,1,1,1,1,1,1,1,17,1.0,1.0,2.0,1500,1200,4,4,1.0,1.176,0.85,,,,,,,,,,5,1.0,6,4.0,62.0,,44.0,4.0,,2.0,5.0,1,1,1,1,1.0,1
13,6,14.0,male,1.0,50,800,11,northern_ireland,E12000006,11,0.9,1.1,,1,1,1,1,1,1,1,2,1.0,1.0,,600,700,2,2,1.0,1.0,1.0,,,,,,,,,,6,1.0,6,4.0,,6.0,,,4.0,,,1,1,1,1,,1
14,6,15.0,male,0.0,50,800,12,northern_ireland,E12000006,12,0.9,1.1,,1,1,1,1,1,1,1,2,1.0,1.0,,600,700,2,2,1.0,1.0,1.0,,,,,,,,,,6,1.0,6,4.0,,6.0,,,4.0,,,1,1,1,1,,1
15,7,16.0,male,0.0,50,800,13,england,E12000007,13,0.9,1.1,,1,1,1,1,1,1,1,17,,,1.0,1500,1200,5,1,1.3,1.176,1.105,,,,,,,,,,7,1.0,6,4.0,90.0,,64.0,4.0,,1.0,7.0,1,1,1,1,1.0,1
16,7,17.0,male,0.0,50,800,13,england,E12000007,13,0.9,1.1,,1,1,1,1,1,1,1,17,,,1.0,1500,1200,5,1,1.3,1.176,1.105,,,,,,,,,,7,1.0,6,4.0,90.0,,64.0,4.0,,1.0,7.0,1,1,1,1,1.0,1
17,7,18.0,male,0.0,50,800,13,england,E12000007,13,0.9,1.1,,1,1,1,1,1,1,1,17,1.0,1.0,1.0,1500,1200,5,1,1.3,1.176,1.105,,,,,,,,,,7,1.0,6,4.0,90.0,,64.0,4.0,,1.0,7.0,1,1,1,1,1.0,1
18,7,19.0,male,0.0,50,800,13,england,E12000007,13,0.9,1.1,,1,1,1,1,1,1,1,17,,,1.0,1500,1200,5,1,1.3,1.176,1.105,,,,,,,,,,7,1.0,6,4.0,90.0,,64.0,4.0,,1.0,7.0,1,1,1,1,1.0,1
19,7,20.0,male,0.0,50,800,13,england,E12000007,13,0.9,1.1,,1,1,1,1,1,1,1,17,,,1.0,1500,1200,5,1,1.3,1.176,1.105,,,,,,,,,,7,1.0,6,4.0,90.0,,64.0,4.0,,1.0,7.0,1,1,1,1,1.0,1
